### PR TITLE
Repository functions to manipulate answer reactions

### PR DESCRIPTION
**Pull request by axross**

### CODE REVIEW

**Comment by axross** [comment]
@ryotogashi Can you have a look? Thanks!

**Comment by axross** [comment]
@ryotogashi ping

File: repositories/createAnswerReaction.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export function createCreateAnswerReaction({
+  firebaseApp
+}: {
+  firebaseApp: firebase.app.App;
+}): CreateAnswerReaction {
+  return async ({ answer, reactionType, user }) => {
+    await firebaseApp
+      .firestore()
+      .collection("post")
```
Comment (ryotogashi) [inline repositories/createAnswerReaction.ts]: posts?

Comment (axross) [reply repositories/createAnswerReaction.ts]: That's right. Thanks! I really wanna write tests for that...

**Comment by axross** [comment]
@ryotogashi Fixed. Could you have a look again just in case?

**Comment by ryotogashi** [comment]
@axross I was actually wondering why we don't use cloud function for manipulating answer reaction. Other than that looks good to me!

**Comment by axross** [comment]
@ryotogashi 

> I was actually wondering why we don't use cloud function for manipulating answer reaction.

I don't get what you meant. Why do we should use cloud functions to add/remove reactions to answers?

**Pull request merged**